### PR TITLE
Expose Mesh get_aabb

### LINKED ISSUE #33535: GDNative - get_aabb on mesh is not exposed
**Godot version:**
3.2 
**OS/device including version:**
Any OS
**Issue description:**
Normally this would be a feature request, but the absence of this function makes it hard to achieve some simple functionalities.

Example: If you want to have meshes, that are used for internal calculations instead of making them visible - e.g. navigation meshes. You can't access the get_aabb function on ArrayMesh/PrimitiveMesh. get_aabb function is already public in source code, but it's not exposed. Instead you have to create a MeshInstance, to get the aabb function. But the MeshInstance should be present in the tree (otherwise I get an error), which is not what one would want with internal meshes, and it also creates overhead, as all the needed functionalities are already in AMesh/PMesh. So currently I have no other option than to create a separate instance of AABB and set it as custom (which is exposed for some reason). But that duplicates the AABB creation logic which is lame.

EDIT: Since it's quite specific issue (not a lot of people use GDNative) and also looks like a simple one, I can volounteer to create a PR, just need a green light, or an explanation if a fix is not wanted.

### PR DESCRIPTION
**Pull request by aaronfranke**
No clue why this wasn't done before. Fixes #33535

### CODE REVIEW

**Comment by akien-mga** [comment]
Please also add documentation.

And this will raise the question: what information will users get with this method? `Mesh::get_aabb()` seems to be a virtual and always 0 in the base `Mesh`, and it's only overridden by Mesh-derived classses: `ArrayMesh` and `PrimitiveMesh`. CC @BastiaanOlij

**Comment by bojidar-bg** [comment]
Also, what happens when the user gives a large `custom_aabb`? Does `get_aabb` still return the smallest AABB?

**Comment by aaronfranke** [comment]
@bojidar-bg It appears that it doesn't. The method is just `return aabb;`, and the `set_custom_aabb` method doesn't set `aabb` it sets another field called `custom_aabb`.

File: doc/classes/Mesh.xml [hunk 1]
```diff
@@ -40,6 +40,13 @@
 				Generate a [TriangleMesh] from the mesh.
 			</description>
 		</method>
+		<method name="get_aabb" qualifiers="const">
+			<return type="AABB">
+			</return>
+			<description>
+				Returns the smallest AABB enclosing this Mesh.
```
Comment (akien-mga) [inline doc/classes/Mesh.xml]: Should either be "mesh" (lower case, English word) or "[Mesh]" (class name, with hyperlink - or here, code markup, as it's the class being described).

Comment (akien-mga) [reply doc/classes/Mesh.xml]: I'd also add a "[b]Note:[/b]" that the method is only implemented for [ArrayMesh] and [PrimitiveMesh].

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**